packages/KSelect/KSelectItem.vue

```diff
@@ -81,6 +81,19 @@ export default {
     .k-select-item-label {
       width: auto;
       line-height: 16px;
+      font-weight: 500;
+      font-size: 14px;
+      padding: 8px;
+
+      .select-item-label {
+        font-weight: 600;
```

Comment:
fix: can we add the color here (I forgot to include in the ticket)
```suggestion
        color: var(--grey-500);
        font-weight: 600;
```

Reply:
Updated